### PR TITLE
feat(release notes): add 7.9.1 content

### PR DESCRIPTION
**Pull request by NathalieC**
* add bug fixes for 7.9.1

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -362,8 +362,34 @@ The BonitaSubscription-x.y.z-deploy.zip is no longer provided starting from Boni
...
+* UID-105 When name of artifact is too long, left panel is broken
+
+#### Fixes in Web/Portal component
+* BPO-72 BS-19311 Administration profile: Monitoring page generates a 26 seconds request (search process definition)
+* BPO-106 BS-19310 Administration profile: Open cases page generates twice the same requests (search)
+* BPO-146 No tooltips on the top right icons and name of the user
+* BPO-154 Clean and Improve Layout css file
+* BPO-159 "All profiles" should be the default option in the application list filter of Bonita layout
+* BPO-163 "..." of ellipsis should not be displayed on a white background
+* BPO-167 Avoid user details modal overflow DONE
```
Comment (julienmege) [inline md/release-notes.md]: need to remove the "DONE" line 390 and 392

File: md/release-notes.md [hunk 2]
```diff
@@ -362,8 +362,34 @@ The BonitaSubscription-x.y.z-deploy.zip is no longer provided starting from Boni
...
+
+#### Fixes in UI Designer component
+* UID-83 Message toast in UI Designer is not clear when license is expired
+* UID-104 Fragment variable value isn't clean when variable scope change
+* UID-105 When name of artifact is too long, left panel is broken
+
+#### Fixes in Web/Portal component
+* BPO-72 BS-19311 Administration profile: Monitoring page generates a 26 seconds request (search process definition)
+* BPO-106 BS-19310 Administration profile: Open cases page generates twice the same requests (search)
+* BPO-146 No tooltips on the top right icons and name of the user
```
Comment (julienmege) [inline md/release-notes.md:386]: maybe we should reword with "Add Tooltips ..." ?

File: md/release-notes.md (hunk 1 above)
Comment (NathalieC) [reply md/release-notes.md]: done

File: md/release-notes.md (hunk 2 above)
Comment (NathalieC) [reply md/release-notes.md:386]: bug names are about problems, not solution

**Pull request merged**